src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,48 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
     public static void main(String[] args) {
-        MovieService movieService = null;
+        Injector injector = Injector.getInstance("mate.academy");
 
+        //Creation of Movie
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
+        MovieService movieService =
+                (MovieService) injector.getInstance(MovieService.class);
         movieService.add(fastAndFurious);
         System.out.println(movieService.get(fastAndFurious.getId()));
         movieService.getAll().forEach(System.out::println);
+
+        //Creation of Cinema Hall
+        CinemaHall redHall = new CinemaHall();
+        redHall.setCapacity(100);
+        redHall.setDescription("Red Hall");
+        CinemaHallService cinemaHallService =
+                (CinemaHallService) injector.getInstance(CinemaHallService.class);
+        cinemaHallService.add(redHall);
+        System.out.println(cinemaHallService.get(redHall.getId()));
+        cinemaHallService.getAll().forEach(System.out::println);
+
+        //Creation of Movie Session
+        MovieSession redHallMovieSession = new MovieSession();
+        redHallMovieSession.setMovie(fastAndFurious);
+        redHallMovieSession.setCinemaHall(redHall);
+        redHallMovieSession.setShowTime(LocalDateTime.of(2024, 11, 11, 15, 0));
+        MovieSessionService movieSessionService =
+                (MovieSessionService) injector.getInstance(MovieSessionService.class);
+        movieSessionService.add(redHallMovieSession);
+        System.out.println(movieSessionService.get(redHallMovieSession.getId()));
```

Comment:
Ensure that the `add` method in `MovieSessionService` assigns an ID to `redHallMovieSession` before calling `get`. Otherwise, `redHallMovieSession.getId()` might return null or an invalid ID.